src/components/ExportDialog/ExportDialog.js

```diff
@@ -54,25 +75,23 @@ class ExportDialog extends PureComponent {
       email,
       end,
       getFullTime,
-      handleExportDialogClose,
-      isExportOpen,
+      isOpen,
       loading,
       start,
-      startExport,
       t,
-      type,
       timestamp,
       timezone,
       location,
     } = this.props
     const { currentTargets } = this.state
-    if (!isExportOpen) {
+    if (!isOpen) {
       return null
     }
-    const isWallets = location && location.pathname && getTarget(location.pathname) === queryConstants.MENU_WALLETS
+    const target = getTarget(location.pathname)
+    const isWallets = location && location.pathname && target === queryConstants.MENU_WALLETS
```

Comment:
in line 90 you check for `location.pathname` but in this line you double check for `location`? do we really need it?

Reply:
yes, it's unnecessary, `withRouter` should provide it at all times